templates/patterns/featured_media/pattern-featured-media--variant-inside_text.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+/**
+ * @file
+ * Featured media component.
+ */
+#}
+
+{% if alignment == 'right' %}
+  {% set text_alignment_class = 'order-md-1' %}
+  {% set video_alignment_class = 'order-md-2' %}
+{% else %}
+  {% set text_alignment_class = 'order-md-2' %}
```

Comment:
set these as default and after override if `alignment == 'right'` so we skip the else